NEWS.md

```diff
@@ -1,5 +1,6 @@
 ## rtables 0.6.0.1
  * bumped devel version
+ * Joining 2 tables via `rbind` will retain annotations if only present in one table or shared between both tables.
```

Comment:
We shouldn't use the word "joining" here as it has a specific meaning in data science, which isn't what we're doing.

Also "annotations" isn't specific enough, we would want to say "title and footer information" or something similar.

Reply:
Updated accordingly.